coffee_testing/src/lib.rs

```diff
@@ -40,6 +39,36 @@ pub mod macros {
         };
     }
 
+    #[macro_export]
+    macro_rules! httpd {
+        ($dir:expr, $port:expr, $($opt_args:tt)*) => {
+            async {
+                use std::process::Stdio;
+
+                use tokio::process::Command;
+
+                let opt_args = format!($($opt_args)*);
+                let args = opt_args.trim();
+                let args_tok: Vec<&str> = args.split(" ").collect();
+
+                let cargo_target = std::env::var("CARGO_TARGET_DIR").unwrap();
```

Comment:
this is wrong, I make this working in the https://github.com/coffee-tools/coffee/pull/91

Reply:
```rust
     let cargo_target = concat!(env!("CARGO_MANIFEST_DIR"), "/..");
    let path = std::path::Path::new(cargo_target).to_str().unwrap();
```